less/styles.less

```diff
@@ -82,4 +81,24 @@
 
 .logo-eclipse-default-mobile {
   max-width:165px;
-}
\ No newline at end of file
+}
+
+@media (max-width: 767px) {
```

Comment:
Instead of this block, try the following variables. Had to dig a bit to find them, as they are built into bootstrap rather than our stuff:

@header-toolbar-background-color:transparent;
@mobile-menu-background: transparent;

Reply:
Nice catch!